content/fast/replace-gifs-with-videos/codelab-replace-gifs-with-video.md

```diff
@@ -169,16 +165,15 @@ instead. If you prefer a WebM <code>&lt;source&gt;</code> be used first, specify
 
 -  Click on the **Show Live** button.
 
-<web-screenshot type="show-live"></web-screenshot>
-
 The experience should look the same. So far so good.  
 
 ## Verify with Lighthouse
 
 With the live site open:
-1. Open Chrome DevTools.
+1. Open Chrome DevTools by pressing `CMD + OPTION + i` / `CTRL + SHIFT + i`.
```

Comment:
```suggestion
1. Open Chrome DevTools.
```

I think if we move the instructions to the first step then it may be ok to remove them here.